tests/ui/test/specs/read-s3-csv.spec.ts

```diff
@@ -22,11 +22,12 @@ test.describe('As a user I want to be able to read csv files downloaded from s3
     allure.suite('Tests downloaded files');
     allure.subSuite('Tests csv files');
     await test.step('Navigate to landing page', async () => {
+      await page.goto('/');
     });
```

Comment:
Nitpick - no need for the extra async/await if you return the promise directly
```suggestion
    await test.step('Navigate to landing page', () => page.goto('/'));
```

Reply:
Updated